NAMESPACE

```diff
@@ -128,6 +128,9 @@ export(configure)
 export(execute_array_udf)
 export(execute_generic_udf)
 export(execute_multi_array_udf)
+export(list_arrays)
+export(list_public_arrays)
+export(list_shared_arrays)
```

Comment:
Can you make it `list_arrays(..., public=TRUE, shared=TRUE)` and then just flip the booleans, instead of repeating what from a distance looks like the same function three times?  Or is that a backend feature that we cannnot toggle public/private and shared/non-shared?

Reply:
@eddelbuettel good call and I should have pointed that out.

I'm just imitating the Python API, straight up.

On the other hand, it's not clear to me how many of our users will be simultaneously using Python _and_ R cloud APIs; I'm not sure how important it is to keep that consistency.

@ihnorton and/or @Shelnutt2 may have some thoughts on the matter -- ?

Reply:
+1 for keywords if that is more idiomatic R (I don't have strong opinions on that topic)

Reply:
@ihnorton :  Sorry, what "keywords"?  I am missing something here...

@johnkerl :  You control your API.  You could even put a single wrapper function on top that calls the (most appropriate) REST end point given the user request.  To me, and on the margin, this just seems like _one_ call:  "get me a list of arrays" subject to possibly constraints of only public or shared ones.   But if it doesn't to you so be it....

Reply:
> what "keywords"?

python calls `public=TRUE, shared=TRUE` "keyword args"